install.sh

```diff
@@ -138,9 +138,16 @@ if [[ -n "$conda_pkg_file" ]]; then
   done
 fi
 
-# Due to https://github.com/conda/conda/issues/8089, we have to install pocl-cuda
-# manually on Linux.
-[[ $(uname) == "Linux" ]] && conda install pocl-cuda
+# Due to https://github.com/conda/conda/issues/8089, we have to install these
+# packages manually on specific operating systems:
+
+# Required for Nvidia GPU support on Linux (package does not exist on macOS)
+[[ $(uname) == "Linux" ]] && conda install --yes pocl-cuda
+
+# Required to use pocl on macOS Big Sur
+# (https://github.com/illinois-ceesd/emirge/issues/114)
```

Comment:
```suggestion
# https://github.com/illinois-ceesd/emirge/issues/114
# https://github.com/conda-forge/pocl-feedstock/issues/52
```

Reply:
PR was automerged without this change :-(